pkg/remote/client_cache_test.go

```diff
@@ -0,0 +1,75 @@
+package remote_test
```

Comment:
Could you please maybe add this package to the unit-test-coverage file?

Reply:
Sure, the coverage is very low, because the remote package contains a lot. On another branch concerning [this](https://github.com/kyma-project/lifecycle-manager/issues/1126) refactoring, the package it is moved and nicely structured and also separated in subpackages so the coverage can be adapted then.